docs/book/src/topics/powervs/creating-a-cluster.md

```diff
@@ -123,4 +130,97 @@ following the steps below.
     ibm-powervs-1-control-plane-rg6xv    Ready    master   41h   v1.26.2
     ibm-powervs-1-md-0-4dc5c             Ready    <none>   41h   v1.26.2
     ibm-powervs-1-md-0-dbxb7             Ready    <none>   20h   v1.26.2
-    ```
+
+### Deploy a PowerVS cluster with user provided resources
+
+  ```
+  IBMPOWERVS_SSHKEY_NAME="my-pub-key" \
+  IBMPOWERVS_VIP="192.168.167.6" \
+  IBMPOWERVS_VIP_EXTERNAL="163.68.65.6" \
+  IBMPOWERVS_VIP_CIDR="29" \
+  IBMPOWERVS_IMAGE_NAME="capibm-powervs-centos-streams8-1-26-2" \
+  IBMPOWERVS_SERVICE_INSTANCE_ID="3229a94c-af54-4212-bf60-6202b6fd0a07" \
+  IBMPOWERVS_NETWORK_NAME="capi-test" \
+  clusterctl generate cluster ibm-powervs-1 --kubernetes-version v1.26.2 \
+  --target-namespace default \
+  --control-plane-machine-count=3 \
+  --worker-machine-count=1 \
+  --flavor=powervs | kubectl apply -f -
+  ```
```

Comment:
Can we add more content here on explaining the usage of existing resources and the different combinations user can pass like TG, VPC and PowerVS service instance?

Reply:
Currently the user provided TG expects to have both PowerVS and VPC connections.
1. When connections are not there create the connection and during delete phase clean up only the connection. It requires another field to be added in status to mark whether connection is created by controller or not.
2. We can expect combination of powervs, vpc and Tg to be passed and their connection should already be there.
If we are going with 2nd approach, we need to document that. IMO we can go with second approach and keep things simple for reuse approach. 
Wdyt? @Amulyam24 @Karthik-K-N @mkumatag

Reply:
Yeah, I think its better to  go with second approach and document it.

Reply:
> Currently the user provided TG expects to have both PowerVS and VPC connections.
> 
> 1. When connections are not there create the connection and during delete phase clean up only the connection. It requires another field to be added in status to mark whether connection is created by controller or not.
> 2. We can expect combination of powervs, vpc and Tg to be passed and their connection should already be there.
>    If we are going with 2nd approach, we need to document that. IMO we can go with second approach and keep things simple for reuse approach.
>    Wdyt? @Amulyam24 @Karthik-K-N @mkumatag

I don't think this is relevant with the latest code which supports all combinations of TG. Added a generic comment, PTAL